.github/dependabot.yml

```diff
@@ -0,0 +1,19 @@
+version: 2
+
+updates:
+  - package-ecosystem: "maven"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    allow:
+      - dependency-name: "io.netty:*"
+      - dependency-name: "com.fasterxml.jackson.core:*"
+      - dependency-name: "com.fasterxml.jackson.module:*"
```

Comment:
We can configure using [Jackson bom ](https://github.com/FasterXML/jackson-bom) to keep this as a single declaration in the pom.